time_monkey/settings/production.py

```diff
@@ -0,0 +1,7 @@
+# pylint: disable=unused-wildcard-import
```

Comment:
You gave everywhere pylint disabling but I can't see anywhere that you add pylint in this PR. Please delete them and add it to Pull Request where you will add pylint and others necessary programming tools